### DISCUSSION TITLE
進め方について

### DISCUSSION THREAD

**Discussion by Kenzo-Wada** [vike-ja-support/vike-docs-ja#6] [General]
@usk94 
自由にやっていきたいなと思いつつ、作業内容が被ってconfilctは避けたいので、予め日本語化させるページについてIssueで起票してからそれをやる形にするとスムーズかなと思いましたがどうでしょう？？

**Comment by usk94** [comment]
ありがとうございます！はい、それでいいと思います🙆‍♂️
